### ISSUE TITLE
tour: solutions repository not linked anywhere in the tour

### ISSUE THREAD

**Issue by ALTree** [golang/tour#542]
As far as I can tell, the solutions to the tour exercises in the `/solution` folder are not linked in the tour. Users that run the tour locally from the repo may see the folder, but the ones that visit the online version of the tour won't. 

The solutions repo at https://github.com/golang/tour/tree/master/solutions should be linked somewhere.

Either in the introduction, or in the slide of the first exercise.

**Comment by bgadrian** [comment]
I think the link should be present on all exercises? I presume the users will not open the solutions at the first slide, nor return 10 lessons back for the link (presuming that he remembers there was a link).

**Comment by ALTree** [comment]
> think the link should be present on all exercises? 

This seems excessive. I don't think we want to encourage people to immediately look at the exercise solution, maybe without even trying to solve it. I suspect it may be the reason the solution are not currently linked everywhere. This may be a little excessive, and it could be fixed by adding a small note once that links the url of the solutions.

**Comment by bgadrian** [comment]
Ok, if that is the intent of the exercises make sense. 

I think it contravenes with the scope of the tour, I mean is more important that the users solve the problem, even if is copy paste, more importantly they see how Go code should be written. The fact that they know mathematics, image processing or other exercises is pretty irrelevant to the bigger scope.

Anyway, we could insert a new blank page, [before the first exercise](https://tour.golang.org/flowcontrol/8), with the README from the /solution folder and a link? 

The Notes of the first exercise already has a lot of text, so I don't think it would be a good fit.

**Comment by ALTree** [comment]
>  I mean is more important that the users solve the problem, even if is copy paste

I disagree. The most important thing it that the use *tries* to solve the problem by writing code. The exercises are meant to be slightly more difficult than the code examples in the tour, and the point is to encourage a three step learning experience by 1) introducing a concept 2) showing a simple usage example 3) challenging the user to use the concept in a more difficult context

> Anyway, we could insert a new blank page, before the first exercise, with the README from the /solution folder and a link?

A whole page seems overkill (there are already a few almost empty pages, adding one for something that should be a small note is IMO unnecessary). In the welcome page, it says

> Throughout the tour you will find a series of slides and exercises for you to complete. 

maybe a small note with the link could be appended there?
